### PR TITLE
Update readme for Rocketmap discord

### PR DESCRIPTION
**Pull request by ibuckshot5**
## Description

Update the PokeAlarm readme for the RocketMap discord

## How Has This Been Tested?

No need, just a readme change 😉 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ PokeAlarm is an easy to use yet highly configurable webserver designed to receiv
 ## Looking for Help?
 
 #### Wiki
-Head on over to the [**PokeAlarm Wiki**](https://github.com/kvangent/PokeAlarm/wiki) to find detailed instructions on setting up and configuring PokeAlarm. You can find the table of contents on the right!
+Head on over to the [**PokeAlarm Wiki**](https://github.com/RocketMap/PokeAlarm/wiki) to find detailed instructions on setting up and configuring PokeAlarm. You can find the table of contents on the right!
 
 #### Discord
-Before visiting your discord channel, check both the [Wiki](https://github.com/kvangent/PokeAlarm/wiki) and the [FAQ](https://github.com/kvangent/PokeAlarm/wiki/faq). If you still can't find what you are looking for, try the [**Discord channel**](https://discord.gg/S2BKC7p) - but make sure you read the **#rules** channel or risk getting banned!
+Before visiting your discord channel, check both the [Wiki](https://github.com/RocketMap/PokeAlarm/wiki) and the [FAQ](https://github.com/kvangent/PokeAlarm/wiki/faq). If you still can't find what you are looking for, try the [**Discord channel**](https://discord.gg/RocketMap)
```
Comment (kvangent) [inline README.md]: You missed one to the wiki.

File: README.md [hunk 2]
```diff
@@ -17,13 +17,13 @@ PokeAlarm is an easy to use yet highly configurable webserver designed to receiv
...
 
 #### Github
 Have an idea for a new feature? Think you found a bug? Head over to our [Github](https://github.com/kvangent/PokeAlarm/issues/new) and open an 'Issue' ticket. Make sure to follow the template completly, or else your issue will be closed without comment.
```
Comment (kvangent) [inline README.md]: Just kidding, you missed two.

Comment (kvangent) [inline README.md]: Looks like you missed two.

**Comment by ibuckshot5** [comment]
@kvangent fixed

**Pull request merged**